scripts/release.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+if [ $# -eq 0 ]
+  then
+    echo "Missing argument for the new version to release"
+    exit 1
+fi
+
+git stash
+sed -i "s/## Unreleased/## Unreleased\n\r## $1/g" CHANGELOG.md
+sed -i "s/\"version\": \".*\",/\"version\": \"$1\",/g" package.json
```

Comment:
A few things:

* On macOS, you need to provide an argument to `-i`, which can be an empty string (`-i ""`) (BSD style).
* I believe we only use Linux line endings (`\n`) in our `README.md` and `CHANGELOG.md` files (I see a `^M` in there BTW; we should be careful with that). I would use `\n\n` instead of `\r\n` (notice I suggest **two** empty lines instead of just one, because that's the pattern we used for all releases in the `CHANGELOG.md` file).
* Finally, the newlines don't work with `sed` in BSD 😑 (hence, not on macOS either). Perhaps we can use something like this:
  ```bash
  sed "s/## Unreleased/## Unreleased__NEW_LINE__## $1/g" CHANGELOG.md | awk '{ sub(/__NEW_LINE__/,"\n\n"); print }' > CHANGELOG.md.back
  mv CHANGELOG.md.back CHANGELOG.md
   ```
  (redirect to `CHANGELOG.md.back`, as you're normally not allowed to update a file your reading from in one go).

Putting all this together:

```bash
sed "s/## Unreleased/## Unreleased__NEW_LINE__## $1/g" CHANGELOG.md | awk '{ sub(/__NEW_LINE__/,"\n\n"); print }' > CHANGELOG.md.back
mv CHANGELOG.md.back CHANGELOG.md
sed -i "" "s/\"version\": \".*\",/\"version\": \"$1\",/g" package.json
```

Reply:
Thanks for the suggestions! :)

Reply:
I couldn't make it work on linux with the (`-i ""`) so I did it with a back file like in the first sed.